mne_bids/tests/test_read.py

```diff
@@ -23,8 +23,9 @@
     import mne
 
 from mne.io import anonymize_info
-from mne.utils import _TempDir, requires_nibabel, check_version, object_diff
+from mne.utils import _TempDir, requires_nibabel, check_version
 from mne.datasets import testing, somato
+from mne.transforms import _str_to_frame
```

Comment:
if it's just a tiny dictionary, please just copy it over. I think it's more legible that way + you don't risk breaking `mne-bids` when there is a new release of `mne-python`.